lib/linguist/languages.yml

```diff
@@ -3475,6 +3475,17 @@ UnrealScript:
   tm_scope: source.java
   ace_mode: java
 
+Ur/Web:
```

Comment:
I was going to modify things to accommodate this but I see it would be better to go with the flow.  I'll swap the primary name and the main alias.  Sorry for the noise.